README.md

```diff
@@ -324,6 +324,12 @@ Ensures that the element is visible to a user. Currently, the following checks a
 - Element has a size (its [bounding box](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) has a non-zero width and height)
 - Element's opacity is greater than 0.05 (opacity of parent elements are considered)
 
+#### Target size
+
+Ensures that the clickable region of an element is large enough to easily be activated by users, even users with reduced fine motor skills or while using a touch input device.
+
+Element must be at least 44px wide and at least 44px tall to pass the target size check (configurable via `user.targetSize` option in `WithBrowserOpts` or `targetSize` option for `user.click`).
```

Comment:
Can we add a link to where these numbers came from? (Is it the Apple HCI guidelines?)

Reply:
We can use this link: https://www.w3.org/TR/WCAG21/#target-size